draft-regext-rdap-x-media-type.md

```diff
@@ -189,11 +189,36 @@ Provisional Registration: No
 
 # Acknowledgements
 
-Pawel Kowalik provided ideas and feedbacks that have contributed to
+Pawel Kowalik and James Mitchel have provided ideas and feedbacks that have contributed to
 the content of this document.
 
 {backmatter}
 
+# Using the Vary Header {#vary_header}
+
+Server implementers may want to consider using the `vary` header depending on the caching
+behavior desired of shared caches (i.e. middleboxes, not client caches).
+
+Consider the following scenario where user Bob and user Alice send queries to the same
+RDAP server that is routed through a middlebox network element implementing a shared HTTP caceh.
+
+User Bob sends a query for the domain `example.com`
+(http://regy.example/domain/example.com) without RDAP-x. The `accept` header sent
+for Bob's query would be `accept: application/rdap+json` or `accept: application/json`.  
+User Alice sends a query for the same domain, however her client uses RDAP-X. The `accept`
```

Comment:
How about "User Alice later sends" since we below consider the temporal order of queries ("return the answer of the first query to the second query")?

Reply:
done